Sofa/Component/SolidMechanics/FEM/Elastic/src/sofa/component/solidmechanics/fem/elastic/TetrahedronFEMForceField.h

```diff
@@ -199,7 +198,7 @@ class TetrahedronFEMForceField : public BaseLinearElasticityFEMForceField<DataTy
     core::objectmodel::RenamedData<std::string> f_method;
 
     SOFA_ATTRIBUTE_DEPRECATED__RENAME_DATA_IN_SOLIDMECHANICS_FEM_ELASTIC()
-    core::objectmodel::RenamedData<Real> _poissonRatio;
+    DeprecatedAndRemoved _poissonRatio;
```

Comment:
sure for this one? Seems to be still a RenamedData.
Otherwise, the _youngModulus need to be changed as well

Reply:
Both are different actually. `_youngModulus` is just a rename, but `_poissonRatio` is a rename + a change of type. So `RenameData` does not apply